launch/includes/rosgraph_nodes.launch.xml

```diff
@@ -1,8 +1,8 @@
 <launch>
 
-  <rosparam ns="rosgraph_manipulator" command="load" file="$(find metacontrol_sim)/yaml/rosgraph_manipulator_config.yaml"/>
+  <rosparam ns="rosgraph_manipulator" command="load" file="$(find mc_rosgraph_manipulator)/config/rosgraph_manipulator_config.yaml">
```

Comment:
Seems fine, just a question.

Why did you remove the `/` from the tag.  I'm not sure if this will work correctly that way

Reply:
Ops, just a typo. I will correct it.